Pipfile.lock

```diff
@@ -1,7 +1,7 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "a36bd6a6121923c3fb9fc26d0c54e116212ff6e2639249f33cd277d4abf983ff"
+            "sha256": "1715e6a6f19caee2af4f02c4e04ec9ef2a383b3c94dc39b72e2abec903f14893"
```

Comment:
we shouldn't have Pipfile.lock updates without Pipfile updates

Reply:
@edyesed Do you know what the command is to install without updating the lock file? I felt weird about it updating, but since it was generated from the Pipfile and that was not updated I figured it was fine. But I can absolutely revert this change :)